tests/unit/backend/test_headers/header2.cpp

```diff
@@ -0,0 +1,12 @@
+#ifndef HEADER_2
+#define HEADER_2
+
+int f1(int x) {
+	return x;
+}
+
+string f2(string s) {
```

Comment:
std::string?

Reply:
Yes @etejedor . ROOT takes the `std` namespace by default right ?

Reply:
Yes that's correct, but it does not harm that your headers are complete (include + namespace) :D

Reply:
Sure @etejedor . Will change this 👍